src/config/settings.py

```diff
@@ -267,6 +267,9 @@ def alt_or_cmd(self) -> str:
     # Mirage must be connected to a terminal for this to work.
     python_debugger = ["Shift+F1"]
 
+    # Quit Mirage
+    # quit = ["Ctrl+Q"]
```

Comment:
```suggestion
    # Quit Mirage
    quit = []
```
The code assumes settings always exist (i.e. they're uncommented and assigned to a value of the right type) at least in the base file, you get an error in the terminal about this otherwise. 
This setting should also be moved after the next one, instead of being in the middle of the two debugger keybinds.